### PR TITLE
PXB-3067 Release notes for PXB 8.0.33-28 (feature release)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   docs/release-notes.md
	new file:   docs/release-notes/8.0/8.0.33-28.0.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/8.0/8.0.33-28.0.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Percona XtraBackup 8.0.33-28 (2023-07-)
+
+[Percona XtraBackup (PXB)](https://www.percona.com/software/mysql-database/percona-xtrabackup) is a 100% open source backup solution for all versions of Percona Server for MySQL and MySQL® that performs online non-blocking, tightly compressed, highly secure full backups on transactional systems. Maintain fully available applications during planned maintenance windows with Percona XtraBackup.
+
+## Release highlights
```
Comment (satya-bodapati) [inline docs/release-notes/8.0/8.0.33-28.0.md:5]: I would add a note that that dictionary cache improves the performance of xtrabackup --prepare

**Review by satya-bodapati — CHANGES_REQUESTED** [review]

File: docs/release-notes/8.0/8.0.33-28.0.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.33-28.0.md:5]: Added

**Review by satya-bodapati — APPROVED** [review]

File: docs/release-notes/8.0/8.0.33-28.0.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+This release includes improvements, features, and bug fixes. Percona has implemented a two-stage release process for each version. The first release primarily ensures compatibility with the latest MySQL version, to help those customers who need an updated version of Percona XtraBackup as soon as possible. The second release contains additional bug fixes and any improvements or new features.
+
+Percona XtraBackup 8.0.33-28 implements a new datasink that uses `FIFO` (named pipes). With the [FIFO datasink](..//..//xbcloud-binary-fifo-datasink.md) feature, you can benefit from faster backups by streaming data in parallel from XtraBackup to `xbcloud`. This feature is in [tech preview](..//..//glossary.md#tech-preview).
```
Comment (satya-bodapati) [inline docs/release-notes/8.0/8.0.33-28.0.md]: We should clarify who will benefit from this feature. I feel that this statement claims that everyone would benefit from faster streaming.

We should clarify that it is intended for LAN usage that has S3 compatible storages like MinIO etc (marce has written about them in blog post).

**Review by satya-bodapati — CHANGES_REQUESTED** [review]

File: docs/release-notes/8.0/8.0.33-28.0.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.33-28.0.md]: updated

Comment (satya-bodapati) [reply docs/release-notes/8.0/8.0.33-28.0.md]: @altmannmarcelo can you please check the wording here? should we say Local Object Storage in LAN or S3 compatible object store in LAN?

File: docs/release-notes/8.0/8.0.33-28.0.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## Release highlights
+
+This release includes improvements, features, and bug fixes. Percona has implemented a two-stage release process for each version. The first release primarily ensures compatibility with the latest MySQL version, to help those customers who need an updated version of Percona XtraBackup as soon as possible. The second release contains additional bug fixes and any improvements or new features.
+
+Percona XtraBackup 8.0.33-28 implements a new data sink that uses the first in, first out (FIF0) method. With the [FIFO data sink](..//..//xbcloud-binary-fifo-datasink.md) feature, users with a Local Area Network (LAN) and streaming capacity of 10Gbps can benefit from faster backups by streaming data in parallel to object storage compatible with Amazon S3. This feature is in [tech preview](..//..//glossary.md#tech-preview).
+
+Percona XtraBackup 8.0.33-28 implements the [Dictionary cache](..//..//dictionary-cache.md) feature. Dictionary cache improves the performance of xtrabackup `--prepare` phase.
```
Comment (satya-bodapati) [inline docs/release-notes/8.0/8.0.33-28.0.md]: s/phase/operation.

File: docs/release-notes/8.0/8.0.33-28.0.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+## Release highlights
+
+This release includes improvements, features, and bug fixes. Percona has implemented a two-stage release process for each version. The first release primarily ensures compatibility with the latest MySQL version, to help those customers who need an updated version of Percona XtraBackup as soon as possible. The second release contains additional bug fixes and any improvements or new features.
+
+Percona XtraBackup 8.0.33-28 implements a new data sink that uses the first in, first out (FIF0) method. With the [FIFO data sink](..//..//xbcloud-binary-fifo-datasink.md) feature, users with a Local Area Network (LAN) and streaming capacity of 10Gbps can benefit from faster backups by streaming data in parallel to object storage compatible with Amazon S3. This feature is in [tech preview](..//..//glossary.md#tech-preview).
```
Comment (altmannmarcelo) [inline docs/release-notes/8.0/8.0.33-28.0.md]:
```suggestion
Percona XtraBackup 8.0.33-28 implements a new data sink that uses the first in, first out (FIF0) method. With the [FIFO data sink](..//..//xbcloud-binary-fifo-datasink.md) feature, users with a streaming capacity of 10Gbps ( typically on a Local Area Network (LAN)) can benefit from faster backups by streaming data in parallel to object storage. This feature is in [tech preview](..//..//glossary.md#tech-preview).
```

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: docs/release-notes/8.0/8.0.33-28.0.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.33-28.0.md]: changed to operation

File: docs/release-notes/8.0/8.0.33-28.0.md (hunk 4 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.33-28.0.md]: updated

**Review by altmannmarcelo — APPROVED** [review]
LGTM

**Review by satya-bodapati — APPROVED** [review]

**Review by mohitj1988 — APPROVED** [review]

**Review by patrickbirch — APPROVED** [review]
LGTM

**Pull request merged**